### DISCUSSION TITLE
Passing `--without-view` flag to spack

### DISCUSSION THREAD

**Discussion by jbadwaik** [GoogleCloudPlatform/ramble#416] [Q&A] [answered]
I'm currently trying to use ramble on easybuild systems and as a first step, I'm thinking of using the `--without-view` option for `spack env create` to use the environment variables set by easybuild. Once that setup works, then I can start exploring the next steps. 

However,  I am not sure how to pass this environment variable to spack. Is there anything I can set in a `yaml` file to help the process along? Thanks

**Comment by douglasjacobsen** [comment, marked as answer]
@jbadwaik I'd be really interested in ways we could support easybuild better. So, if you have any suggestions please let us know.

As an aside, I just pushed a draft PR (https://github.com/GoogleCloudPlatform/ramble/pull/417) where you can `ramble config add 'config:spack:env_create:flags:"--without-view"`

You can even set this within the workspace config if you want, i.e.

```
ramble:
  config:
    spack:
      env_create:
        flags: --without-view
```

Let me know if this helps you out.

**Reply by jbadwaik** [reply]
@douglasjacobsen Thanks a lot for the PR and the support. We are now trying this out along with a couple of other things and will let you know how it goes.  In the meantime, I'm also thinking of what might be helpful to use `ramble` directly with `spack`

**Reply by douglasjacobsen** [reply]
No problem. Feel free to let us know if you need any more help. :)
